content/module-reference/utility-modules/shared/tagging.md

```diff
@@ -55,16 +66,16 @@ The _tag dictionary_ section displays all of the tags that are available in dark
 
 - a check mark [✓] indicates that the tag is attached to all of the selected images
 
-- a minus sign [--] indicates that the tag is attached to at least one of the selected images. If the symbol is next to a category name in the hierarchical _tree_ view, it means that one of the child tags under that category is attached to at least one of the selected images.
+- a minus sign [--] indicates that the tag is attached to at least one of the selected images. If the symbol is next to a node name in the hierarchical _tree_ view, it means that one of the child tags under that node is attached to at least one of the selected images.
 
-- if no indicator symbol is present, this means that the tag is not attached to any of the selected images, or that the category has no child tags attached to any of the selected images.
+- if no indicator symbol is present, this means that the tag is not attached to any of the selected images, or that the node has no child tags attached to any of the selected images.
 
-In the hierarchical _tree_ view, a name in italics represents a category, and not a child tag. Note that categories are just used to organise the tags, and that a category cannot be directly attached to an image or exported.
+In the hierarchical _tree_ view, a name in italics represents a category (a node which not a tag or a tag set as category). Note that nodes are just used to organise the tags, and that a node which is not a tag cannot be directly attached to an image.
```

Comment:
I'm not sure that the text in the brackets is clear. Are you saying that a category is "**either** a node that is not a tag, **or** a tag that has been explicitly defined as a category"? If so, I think it should be stated that way (with both the "either" and the "or" for clarity). 

In the second sentence, where you say "a node which is not a tag" are you just trying to avoid saying the word "category"? or are you explicitly excluding the second definition of "category" from the first sentence?

Reply:
> with both the "either" and the "or" for clarity

I'll do.

> where you say "a node which is not a tag" are you just trying to avoid saying the word "category" or are you explicitly excluding the second definition of "category" from the first sentence ?


I don't how to say another way, but yes, a node ~~which~~ that is not tag (which has also the category property, but the fact it cannot be attached doesn't come from its category property but from the absence of tag).

>  The phrase "a node that is not a tag" defines a particular type of node (a subset of all nodes).

That's the case. I'll use "that" instead of "which" then.

Thanks

Reply:
> I don't how to say another way, but yes, a node which that is not tag (which has also the category property, but the fact it cannot be attached doesn't come from its category property but from the absence of tag).

I'm still not entirely sure I understand. Perhaps I need to do some more reading. It doesn't help that I don't use tagging for my images.

Reply:
I've tried to use properly which and that ... not yet completely sure though  :)
I've reshaped a bit the critical paragraph. I hope that's better now.